mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/VeilederJoyrideRoutes.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.mulighetsrommet.api.routes.v1
+
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import no.nav.mulighetsrommet.api.domain.dto.HarFullfortJoyrideRequest
+import no.nav.mulighetsrommet.api.domain.dto.VeilederJoyrideDto
+import no.nav.mulighetsrommet.api.domain.dto.VeilederJoyrideRequest
+import no.nav.mulighetsrommet.api.plugins.getNavIdent
+import no.nav.mulighetsrommet.api.services.VeilederJoyrideService
+import org.koin.ktor.ext.inject
+
+fun Route.veilederJoyrideRoutes() {
+    val veilederJoyrideService: VeilederJoyrideService by inject()
+
+    route("/api/v1/internal/joyride") {
+        post("lagre") {
+            val request = call.receive<VeilederJoyrideRequest>()
+            call.respond(
+                veilederJoyrideService.save(
+                    VeilederJoyrideDto(
+                        navident = getNavIdent(),
+                        fullfort = request.fullfort,
+                        type = request.type,
+                    ),
+                ),
+            )
+        }
+
+        post("harFullfort") {
```

Comment:
Dette kunne evt bare vært en GET mot `/joyride/{type}`? (Tolket først at det var endepunkt for å mutere`harFullført`)